Mathlib/Order/SuccPred/CompleteLinearOrder.lean

```diff
@@ -122,24 +117,23 @@ alias csSup_mem_of_not_isSuccLimit' := csSup_mem_of_not_isSuccPrelimit'
 /-- See `exists_eq_ciSup_of_not_isSuccPrelimit` for the
 `ConditionallyCompleteLinearOrder` version. -/
 lemma exists_eq_ciSup_of_not_isSuccPrelimit'
-    (hf : BddAbove (Set.range f)) (hf' : ¬ IsSuccPrelimit (⨆ i, f i)) :
-    ∃ i, f i = ⨆ i, f i :=
+    (hf : BddAbove (range f)) (hf' : ¬ IsSuccPrelimit (⨆ i, f i)) : ∃ i, f i = ⨆ i, f i :=
   csSup_mem_of_not_isSuccPrelimit' hf hf'
 
 @[deprecated exists_eq_ciSup_of_not_isSuccPrelimit' (since := "2024-09-05")]
 alias exists_eq_ciSup_of_not_isSuccLimit' := exists_eq_ciSup_of_not_isSuccPrelimit'
 
-lemma IsLUB.mem_of_not_isSuccPrelimit (hs : IsLUB s x) (hx : ¬ IsSuccPrelimit x) :
-    x ∈ s := by
-  obtain (rfl|hs') := s.eq_empty_or_nonempty
+lemma IsLUB.mem_of_not_isSuccPrelimit (hs : IsLUB s x) (hx : ¬ IsSuccPrelimit x) : x ∈ s := by
+  obtain rfl | hs' := s.eq_empty_or_nonempty
   · simp [show x = ⊥ by simpa using hs, isSuccPrelimit_bot] at hx
   · exact hs.mem_of_nonempty_of_not_isSuccPrelimit hs' hx
 
 @[deprecated IsLUB.mem_of_not_isSuccPrelimit (since := "2024-09-05")]
 alias IsLUB.mem_of_not_isSuccLimit := IsLUB.mem_of_not_isSuccPrelimit
 
-lemma IsLUB.exists_of_not_isSuccPrelimit (hf : IsLUB (Set.range f) x) (hx : ¬ IsSuccPrelimit x) :
-    ∃ i, f i = x := hf.mem_of_not_isSuccPrelimit hx
+lemma IsLUB.exists_of_not_isSuccPrelimit (hf : IsLUB (range f) x) (hx : ¬ IsSuccPrelimit x) :
+    ∃ i, f i = x :=
+  hf.mem_of_not_isSuccPrelimit hx
```

Comment:
According to the style guide using a single line is OK for a short proof

Reply:
I also think the current way is fine.